h2o-r/h2o-package/R/explain.R

```diff
@@ -1416,7 +1416,7 @@ handle_ice <- function(model, newdata, column, target, centered, show_logodds, s
                        data = results)
   column_value <- as.data.frame(newdata[[column]])
   histogram <- stat_count_or_bin(!.is_continuous(newdata[[column]]),
-                                 ggplot2::aes(x = .data[[col_name]], y = (.data$..count.. / max(.data$..count..)) * diff(y_range) / 1.61),
+                                 ggplot2::aes(x = .data[[col_name]], y = (ggplot2::after_stat(count) / max(ggplot2::after_stat(count))) * diff(y_range) / 1.61),
```

Comment:
Deprecated (https://github.com/h2oai/h2o-3/issues/15651)
This change requires ggplot2 3.3.0 and above which is already required for PDP and ICE plots so no need to change the required version of ggplot2. (All other plots should work with v3.0.0.)